cloud-platform/add-secrets-to-deployment.md

```diff
@@ -0,0 +1,170 @@
+---
+category: cloud-platform
+expires: 2018-01-31
+---
+# Adding a secret to an application
+
+## Overview
+
+The aim of this guide is to walkthrough the process of adding a secret (in this example for aws access-key credentials) to a previously deployed application in the Cloud Platform.
+
+## Prerequisites
+
+This guide assumes the following:
+
+* You have previously set up an env. See [Creating-a-Cloud-Platform-Environment](https://github.com/ministryofjustice/cloud-platform-user-docs/blob/master/cloud-platform/env-create.md/#using-secrets-as-environment-variables/#creating-a-cloud-platform-environment)
+* You have previously deployed your application. See [Deploying-an-application-to-the-Cloud-Platform](https://github.com/ministryofjustice/cloud-platform-user-docs/blob/master/cloud-platform/app-deploy.md/#deploying-an-application-to-the-cloud-platform)
+* Check your deployment is running. See [Interacting-with-the-application](https://github.com/ministryofjustice/cloud-platform-user-docs/blob/master/cloud-platform/app-deploy.md/#interacting-with-the-application)
+
+## Configuring secrets
+
+The following is an example of encoding (configuring) aws access-key credentials in your deployment.
+See [kuberenetes-using-secrets-as-environment-variables](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables)
+
+for detailed information regarding providing base64 values in secret objects to Kuberenetes pods)
+
+Create your AWS Credentials access key (making a note of the aws_access_key_id and aws_secret_access_key)
+
+put the aws key secret and id into a file (in this example demo-v9.k8s) as follows ('thekey' referencing the aws_access_key_id
```

Comment:
I expect it was because there were prior v1-v8 but it is a little weird to me that this file is called "demo-v9.k8s" (and that the other names are "demo-v9" something). Not a biggie but maybe worth changing to just plain "demo" or maybe even better "secrets-demo" or something more descriptive.